libgalois/include/katana/PropertyGraph.h

```diff
@@ -842,6 +903,633 @@ CreateSymmetricGraph(PropertyGraph* pg);
 KATANA_EXPORT Result<std::unique_ptr<PropertyGraph>>
 CreateTransposeGraphTopology(const GraphTopology& topology);
 
+// TODO(amber): add NodeLabelVecT
+// template <typename NodeLabelVecT, typename EdgeLabelVecT>
+
+class KATANA_EXPORT EdgeLabelIndex : public GraphTopologyTypes {
+  using EdgeLabelTy = PropertyGraph::TypeSetID;
```

Comment:
This type is referenced in the public API via `label`. So, it would be nice for this to be part of the public API as well.

Can you call this something like `EdgeLabel`? The `Ty` suffix is confusing to most as these are just types just like any other type.

Reply:
done.